Foundation/Thread.swift

```diff
@@ -270,6 +274,9 @@ open class Thread : NSObject {
     }
 
     open class var callStackSymbols: [String] {
+        #if os(Android)
```

Comment:
Same nit here about `#if`, `#else` and `#endif`; please also de-indent `return []` by four spaces so that it aligns with `return backtraceAddresses`.